src/components/burger-ingredients/burger-item/burger-item.jsx

```diff
@@ -0,0 +1,28 @@
+import styles from "./burger-item.module.css";
+import { CurrencyIcon } from "@ya.praktikum/react-developer-burger-ui-components";
+import { data } from "../../../utils/data";
+import { Counter } from "@ya.praktikum/react-developer-burger-ui-components";
+
+function BurgerItem({ productId }) {
```

Comment:
**Нужно исправить**

Нужно посмотреть все компоненты и **типизировать** их `пропсы`, если где-то пропустили.

Reply:
Я же правильно понимаю, что тут по сути проверка только одного пропса?
```suggestion
function BurgerItem({ productId }) {...}

BurgerItem.propTypes = {
  productId: PropTypes.string
}
```

Reply:
Правильно ли я понимаю, что тут проверка будет единственная и вот такая?
```suggestion
function BurgerItem({ productId }) {...}

BurgerItem.propTypes = {
 productId: PropTypes.string
}
```

Reply:
Да, верно

Сейчас у Вас только 1 компонент принимает пропсы, его и нужно типизировать